pootle/static/js/common.js

```diff
@@ -203,6 +205,16 @@ PTL.common = {
       });
     });
 
+
+    /* Bind hotkeys */
+    const hotkeys = mousetrap(document.body);
+
+    hotkeys.bind('f1', (e) => {
```

Comment:
Nothing wrong with that being an alternative shortcut. I added it in 237982bda9d88ee0c88500f7c5a2ae396e29e549